ocaml/utils/target_system.mli

```diff
@@ -9,6 +9,10 @@ type architecture =
 
 val architecture : unit -> architecture
 
+val is_64_bit : bool
+
+val is_32_bit : bool
```

Comment:
Why do we need these additional functions when using `Arch.size_int` and other things already work ?

Reply:
Well there were two equivalents in `To_cmm_helpers` previously, and I don't think there is any other value that tells us the answer immediately.  We shouldn't be doing things like `when Arch.size_int = 4` which aren't robust.